pkg/npm/npm.go

```diff
@@ -14,7 +14,9 @@ import (
 )
 
 const (
-	npmBomFilename = "bom-npm.xml"
+	npmBomFilename          = "bom-npm.xml"
+	cycloneDxPackageVersion = "@cyclonedx/cyclonedx-npm@1.11.0"
```

Comment:
Should we change this to only be the version?

Reply:
They moved from cyclonedx/bom to cyclonedx/cyclonedx-npm, hence added both package and version here.
Do you still think we should only have version?